frontend/packages/akr/public/i18n/fi-FI/translation.json

```diff
@@ -429,7 +429,8 @@
         "cookieBanner": {
           "buttonAriaLabel": "Hyväksy evästeet",
           "buttonText": "Hyväksy",
-          "description": "Tämä sivusto käyttää välttämättömiä evästeitä toimiakseen. Voit lukea lisätietoa evästeistä painamalla",
+          "description": "Tämä sivusto käyttää välttämättömiä evästeitä toimiakseen.",
+          "additionalInfo": "Voit lukea lisätietoa evästeistä painamalla",
```

Comment:
Tätä additionalInfo ei nyt käytetä missään?

Reply:
Joo, ei oo nyt käytössä. Tuossa PR:n kuvauksessa vähän järkeilinkin miksi splittasin nämä avaimet. Voi kyllä hyvin ottaa kokonaan poiskin.

Reply:
Aivan, mitä jos otetaan nyt pois tuo avain, niin kun tätä tehdään eteenpäin ei tarvitse tuota reliikkiä muistella.

Reply:
Joo, tää sopii hyvin.